dotnet/samples/Concepts/Memory/VectorStoreFixtures/VectorStore_QdrantContainer_Fixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Docker.DotNet;
+
+namespace Memory.VectorStoreFixtures;
+
+/// <summary>
+/// Fixture to use for creating a Qdrant container before tests and delete it after tests.
+/// </summary>
+public class VectorStore_QdrantContainer_Fixture : IAsyncLifetime
```

Comment:
Is there any reason to use underscores in class names here?

Reply:
No, I can change it.  The samples all have underscores, making it easier to read their names, but I suppose these are not samples, and they are in their own folder, so I'll just change them.